src/redux/ui/screens/data/handle-cta.ts

```diff
@@ -809,6 +812,27 @@ const handleCtaAction = async (
                     break;
                 }
 
+                case 'swap': {
```

Comment:
please don't add other `case` statements here :), use the new model (check default)